terraform/publish.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+readonly MODULE_PATH="<< parameters.path >>"
+readonly MODULE_NAME="<< parameters.module_name >>"
+readonly VERSION_FILE_PATH="<< parameters.version_file_path >>"
+
+if [[ "$MODULE_PATH" == "" ]]; then
+    echo "module_path parameter must be set"
+    exit 2
+fi
+
+if [[ ! -d "$MODULE_PATH" ]]; then
+    echo "module_path \"$MODULE_PATH\" doesn't exist"
+    exit 2
+fi
+
+if [[ "$MODULE_NAME" == "" ]]; then
+    echo "module_name parameter must be set"
+    exit 2
+fi
+
+if [[ "$VERSION_FILE_PATH" == "" ]]; then
+    echo "version_file_path parameter must be set"
+    exit 2
+fi
+
+if [[ ! -f "$VERSION_FILE_PATH" ]]; then
+  echo "Version file \"$VERSION_FILE_PATH\" doesn't exist"
```

Comment:
should it error here, it cats out the file below?

Reply:
yes it should